guides/common/modules/proc_viewing-multiple-content-view-environment-hosts.adoc

```diff
@@ -0,0 +1,45 @@
+[id="Viewing_multiple_content_view_environment_hosts{context}"]
+= Viewing multiple content view environment hosts
+
+Use this procedure to view hosts with multiple content view environments or multi-environment hosts in the {ProjectWebUI} and Hammer CLI.
+
+.Prerequisites
+Ensure that you have at least one multi-environment host. 
+
+.Procedure
+. In the {ProjectwebUI}, navigate to *Hosts* > *All Hosts*.
++
+[NOTE]
+====
+If you have the *Show new host overview page* setting enabled, the new UI appears by default.
+====
++
+. Click *Manage columns*.
```

Comment:
If you _don't_ have that setting enabled, you still have to click the New UI button.